[2022-04-11 14:56:10,876] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: schedule_data.local_to_gcs_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-04-11 14:56:10,887] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: schedule_data.local_to_gcs_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-04-11 14:56:10,887] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-11 14:56:10,887] {taskinstance.py:1239} INFO - Starting attempt 3 of 3
[2022-04-11 14:56:10,888] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-11 14:56:10,902] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-01-01 00:00:00+00:00
[2022-04-11 14:56:10,913] {standard_task_runner.py:52} INFO - Started process 331138 to run task
[2022-04-11 14:56:10,919] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'schedule_data', 'local_to_gcs_task', 'scheduled__2019-01-01T00:00:00+00:00', '--job-id', '1676', '--raw', '--subdir', 'DAGS_FOLDER/nfl_to_gcs.py', '--cfg-path', '/tmp/tmppnntjhfd', '--error-file', '/tmp/tmp4n5pkfg6']
[2022-04-11 14:56:10,920] {standard_task_runner.py:77} INFO - Job 1676: Subtask local_to_gcs_task
[2022-04-11 14:56:10,992] {logging_mixin.py:109} INFO - Running <TaskInstance: schedule_data.local_to_gcs_task scheduled__2019-01-01T00:00:00+00:00 [running]> on host 4ed3eda454fd
[2022-04-11 14:56:11,049] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-11 14:56:11,080] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=schedule_data
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-01T00:00:00+00:00
[2022-04-11 14:56:11,327] {python.py:175} INFO - Done. Returned value was: None
[2022-04-11 14:56:11,347] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=schedule_data, task_id=local_to_gcs_task, execution_date=20190101T000000, start_date=20220411T145610, end_date=20220411T145611
[2022-04-11 14:56:11,413] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-11 14:56:11,459] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-04-11 19:06:56,667] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: schedule_data.local_to_gcs_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-04-11 19:06:56,683] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: schedule_data.local_to_gcs_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-04-11 19:06:56,683] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-11 19:06:56,683] {taskinstance.py:1239} INFO - Starting attempt 3 of 3
[2022-04-11 19:06:56,683] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-11 19:06:56,704] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-01-01 00:00:00+00:00
[2022-04-11 19:06:56,711] {standard_task_runner.py:52} INFO - Started process 352111 to run task
[2022-04-11 19:06:56,716] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'schedule_data', 'local_to_gcs_task', 'scheduled__2019-01-01T00:00:00+00:00', '--job-id', '1712', '--raw', '--subdir', 'DAGS_FOLDER/nfl_to_gcs.py', '--cfg-path', '/tmp/tmpkzzre1j1', '--error-file', '/tmp/tmp07oc64g5']
[2022-04-11 19:06:56,717] {standard_task_runner.py:77} INFO - Job 1712: Subtask local_to_gcs_task
[2022-04-11 19:06:56,777] {logging_mixin.py:109} INFO - Running <TaskInstance: schedule_data.local_to_gcs_task scheduled__2019-01-01T00:00:00+00:00 [running]> on host 4ed3eda454fd
[2022-04-11 19:06:56,824] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-11 19:06:56,846] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=schedule_data
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-01T00:00:00+00:00
[2022-04-11 19:06:56,849] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'next_ds' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds }}' instead.
[2022-04-11 19:06:56,850] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'next_ds_nodash' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds_nodash }}' instead.
[2022-04-11 19:06:56,851] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'next_execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_end' instead.
[2022-04-11 19:06:56,852] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'prev_ds' from the template is deprecated and will be removed in a future version.
[2022-04-11 19:06:56,853] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'prev_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-04-11 19:06:56,853] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'prev_execution_date' from the template is deprecated and will be removed in a future version.
[2022-04-11 19:06:56,854] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'prev_execution_date_success' from the template is deprecated and will be removed in a future version. Please use 'prev_data_interval_start_success' instead.
[2022-04-11 19:06:56,855] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'tomorrow_ds' from the template is deprecated and will be removed in a future version.
[2022-04-11 19:06:56,856] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'tomorrow_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-04-11 19:06:56,856] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'yesterday_ds' from the template is deprecated and will be removed in a future version.
[2022-04-11 19:06:56,857] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'yesterday_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-04-11 19:06:57,122] {python.py:175} INFO - Done. Returned value was: None
[2022-04-11 19:06:57,135] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=schedule_data, task_id=local_to_gcs_task, execution_date=20190101T000000, start_date=20220411T190656, end_date=20220411T190657
[2022-04-11 19:06:57,169] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-11 19:06:57,205] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
